pyquil/device/_main.py

```diff
@@ -290,7 +290,9 @@ def __init__(self, topology: nx.Graph) -> None:
     def qubit_topology(self) -> nx.Graph:
         return self.topology
 
-    def get_isa(self, oneq_type: str = "Xhalves", twoq_type: str = "CZ") -> ISA:
+    def get_isa(self,
+                oneq_type: str = "Xhalves",
+                twoq_type: Optional[Union[str, List[str]]] = None) -> ISA:
```

Comment:
Why don't you want to provide a default of `twoq_type = ["CZ", "XY"]` here and above?

Reply:
not good to put mutable objects (including lists) into function argument default values in python

Reply:
😬